datastore/db/management/commands/delete_datagetter_data.py

```diff
@@ -27,16 +28,41 @@ def add_arguments(self, parser):
             help="Delete the oldest datagetter data",
         )
 
+        parser.add_argument(
+            "--older-than-days",
+            type=int,
+            help="Delete datagetter data that's more than N days old.",
+        )
+
+        parser.add_argument(
+            "--force-delete-in-use-data",
+            action="store_true",
+            help="Delete datagetter data even if it's in use by a latest best.",
```

Comment:
One tiny thing, can the --oldest help text be amended to say "Delete the oldest datagetter data that isn't in use (unless overridden with --force-delete-in-use-data)" or something to that affect.